slsReceiverSoftware/src/Implementation.cpp

```diff
@@ -962,9 +962,14 @@ void Implementation::setUDPSocketBufferSize(const int64_t s) {
     for (unsigned int i = 0; i < listSize; ++i) {
```

Comment:
This could be size_t

Reply:
actually when I think about it a range based for loop is even better:
```cpp
for (auto& l : listener)
    l.createDummy...
```

Reply:
done